build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -295,6 +310,18 @@
     />
     <Touch Files="@(_InstallMonoPosixHelperOutput);@(_InstallUnstrippedMonoPosixHelperOutput)" />
   </Target>
+  <Target Name="_InstallMonoDoc"
+      Inputs="@(_MonoDocItems);$(_MonoOutputDir)\mdoc.exe"
+      Outputs="@(_MonoDocInstalledItems)">
+    <MakeDir Directories="$(_MandroidDir)" />
+    <Copy
+      SourceFiles="@(_MonoDocItems)"
+      DestinationFolder="$(_MandroidDir)"
+      />
+    <Exec
+      Command="$(RemapAssemblyRefTool) $(_MonoOutputDir)\mdoc.exe $(_MandroidDir)\mdoc.exe Mono.Cecil $(_MandroidDir)\Xamarin.Android.Cecil.dll"
```

Comment:
This should use `&quot;` around paths, if/when the path contains spaces:

    Command="$(RemapAssemblyRefTool) &quot;$(_MonoOutputDir)\mdoc.exe&quot; &quot;$(_MandroidDir)\mdoc.exe&quot; Mono.Cecil &quot;$(_MandroidDir)\Xamarin.Android.Cecil.dll&quot;"